pinot-controller/src/test/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManagerStatelessTest.java

```diff
@@ -540,6 +540,19 @@ public void testValidateTenantConfig() {
     // Valid tenant config without tagOverrideConfig
     _helixResourceManager.validateTableTenantConfig(offlineTableConfig);
 
+    // Invalid serverTag in tierConfigs
+    TierConfig tierConfig = new TierConfig("myTier", TierFactory.TIME_SEGMENT_SELECTOR_TYPE, "10d", null,
+        TierFactory.PINOT_SERVER_STORAGE_TYPE, "Unknown_OFFLINE", null, null);
```

Comment:
add a test for null? (looks like TierConfig json property server tag is nullable)

Reply:
good point. added.